src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -567,12 +568,38 @@ public void onClick(DialogInterface dialog, int which) {
                 builder.create().show();
                 break;
             }
+            case R.id.action_switch_view:{
+                if (isGridView()){
+                    item.setTitle(getApplicationContext().getString(R.string.action_switch_grid_view));
```

Comment:
```getAppicationContext()``` is not really needed. ```FileDisplayActivity``` is a ```Context```, so ```getString``` is available in ```this```.

Or am I missing something?